backend/pkg/httpserver/list_aggregated_feature_support_test.go

```diff
@@ -138,12 +139,13 @@ func TestListAggregatedFeatureSupport(t *testing.T) {
 		{
 			name: "Success Case - include optional params",
 			mockConfig: &MockListBrowserFeatureCountMetricConfig{
-				expectedBrowser:   "chrome",
-				expectedStartAt:   time.Date(2000, time.January, 1, 0, 0, 0, 0, time.UTC),
-				expectedEndAt:     time.Date(2000, time.January, 10, 0, 0, 0, 0, time.UTC),
-				expectedPageSize:  50,
-				expectedPageToken: inputPageToken,
-				err:               nil,
+				expectedTargetBrowser:       "chrome",
+				expectedTargetMobileBrowser: "chrome_android",
```

Comment:
You can use the valuePtr helper we have
```suggestion
				expectedTargetMobileBrowser: valuePtr("chrome_android"),
```